include/cantera/numerics/funcs.h

```diff
@@ -10,6 +10,7 @@
 #define CT_FUNCS_H
 
 #include "cantera/base/ct_defs.h"
+#include "cantera/numerics/polyfit.h"
```

Comment:
polyfit is used by simpsonQuadrature.
```
    for (size_t i = 0; i < ns; i++) {
        double c[3];
        double xs[3] = {x[2*i],x[2*i+1],x[2*i+2]};
        double ys[3] = {y[2*i],y[2*i+1],y[2*i+2]};
        polyfit(3, 2, xs, ys, nullptr, c);
        sum += c[0] * x[2*i+2] +
               c[1] * 0.5 * x[2*i+2] * x[2*i+2] +
               c[2] * 1./3. * x[2*i+2] * x[2*i+2] * x[2*i+2] -
               c[0] * x[2*i] -
               c[1] * 0.5 * x[2*i] * x[2*i] -
               c[2] * 1./3. * x[2*i] * x[2*i] * x[2*i];
    }
```

Reply:
Right, so then the `#include` can be moved to `funcs.cpp` where `simpsonQuadrature` is defined.

The goal is to minimize unnecessary includes in header files because that can slow down compilation. One include doesn't make much of a difference, but the accumulated impact of many unnecessary includes across the whole project can add up.